README.md

```diff
@@ -1,5 +1,6 @@
 # Blockchain Demo
 A web-based demonstration of blockchain concepts.
+_(If you want to read this in Brazilian Portuguese [click this link](docs/pt-br/index.html))_.
```

Comment:
Could be:
_Brazilian Portuguese version: [README.pt-br.md](README.pt-br.md)_
For me, no problem if this reference isn't inside README.md. In this case, no more modifications are needed. It's up to you! 😄 
Do we agree?